AlfrescoAuth/Sources/Model/AlfrescoCredential.swift

```diff
@@ -48,7 +48,11 @@ public struct AlfrescoCredential: Codable {
             case "expires_in":
                 accessTokenExpiresIn = value as? Int
             case "refresh_token":
-                refreshToken = value as? String
+                if let intValue = value as? Int {
```

Comment:
Create a method and return the desired output.

Reply:
Done, Please Review

Reply:
> Create a method and return the desired output.

Done, Please review